tensilelite/Tensile/Tests/common/gemm/dtvA_swizzleA.yaml

```diff
@@ -120,11 +99,296 @@ BenchmarkProblems:
           - Exact: [512,   256, 1, 224]
           - Exact: [512,   256, 1, 256]
           - Exact: [512,   256, 1, 288]
-          - Exact: [127, 127, 1, 127]
-          - Exact: [128, 127, 1, 127]
-          - Exact: [1, 127, 1, 127]
-          - Exact: [127, 1, 1, 127]
         - BiasTypeArgs: ['h']
```

Comment:
Same question, why removing these **4 problems** and **SAV** ? 
BTW, in order to make this 4 problems tested, please help remove the two parameters in this block
```
        - AssertFree0ElementMultiple: [16]
        - AssertSummationElementMultiple: [32]
```

I'm guessing maybe you didn't do rebase?